Jenkinsfile

```diff
@@ -15,7 +16,7 @@ pipeline {
         stage('build') {
             steps {
                 sh '''\
-                  docker build -t "${image_name}" -f k8s.Dockerfile .
+                  docker build --build-arg stats_version=${stats_version} --label stats_version=${stats_version} -t "${image_name}" -f k8s.Dockerfile .
```

Comment:
Annoying that it has to be baked into the build, but I see why it can't be a runtime param atm.

Reply:
elm ftw